charts/vald/values/vald-backup-via-s3.yaml

```diff
@@ -0,0 +1,91 @@
+defaults:
+  image:
+    tag: latest
+
+gateway:
+  lb:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 50Mi
+    gateway_config:
+      # number of index replica.
+      index_replica: 2
+
+agent:
+  minReplicas: 6
+  maxReplicas: 6
+  podManagementPolicy: Parallel
+  resources:
+    requests:
+      cpu: 100m
+      memory: 50Mi
+    limits:
+      memory: 1Gi
+  # Please set a long time to consider the S3 backup time.
```

Comment:
```suggestion
  # We recommend to set this value long enough to ensure the backup speed of S3, since the Index is backed up at the end of the pod.
```